components/Modal.js

```diff
@@ -18,7 +24,7 @@ export default function Modal({ show = false, setShow, title, children }) {
           <div className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" />
         </Transition.Child>
 
-        <div className="fixed inset-0 z-10 overflow-y-auto">
+        <div className={`${modalStyles} fixed inset-0 z-10 overflow-y-auto`}>

```

Comment:
This should come at the end of the line, so that it overwrites previous styles if they are similar

Reply:
Resolved all!